modules/virt-adding-a-boot-source-web.adoc

```diff
@@ -35,7 +35,7 @@ There are four methods for selecting and adding a boot source in the web console
 
 . In the *Add boot source to template* window, click *Select boot source*, select a method for creating a persistent volume claim (PVC): *Upload local file*, *Import via URL*, *Clone existing PVC*, or *Import via Registry*.
 
-.. Optional: Click *Mount this as a CD-ROM boot source* to add an additional disk and mount the disk as a CD-ROM.
+. An additional disk is required for installing the operating system. This additional disk is added automatically and you can customize it later when you are creating the virtual machine. Click *Mount this as a CD-ROM disk* if you want this additional disk to be mounted as a CD-ROM.
```

Comment:
```suggestion
. Optional: Click *Mount this as a CD-ROM boot source* to add an additional disk and mount the disk as a CD-ROM. When mounting a CD-ROM an additional empty disk is added automatically, if you do not need an additinal empty disk when mounting a CD-ROM you can remove later when you create the virtual machine.
```

Usually users will use the CD-ROM to install an operating system on an empty disk, and then remove the CD ROM and use only the disk, If a user mount the boot source as a CD-ROM we assume they want to install an operating system to an empty disk from the CD-ROM , so we automatically add an empty disk mounted as DISK in addition to the CD-ROM that is mounted as CD-ROM

Reply:
@yaacov I've reworded for simplicity and clarity. Please review. Thanks!